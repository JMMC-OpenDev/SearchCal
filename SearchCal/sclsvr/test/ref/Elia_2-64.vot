version https://git-lfs.github.com/spec/v1
oid sha256:e5b818ddc89ad2c453072ae14c7c7f6f98e8cff075fcaffe0ab67a4fe09310aa
size 57388
