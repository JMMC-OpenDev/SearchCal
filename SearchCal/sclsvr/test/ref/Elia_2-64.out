version https://git-lfs.github.com/spec/v1
oid sha256:a181be0086fd9b10f8f3bd21c0c8a522b3992acf09920d3d1e00cecf7d59d772
size 267927
